src/test/gcp/apphosting.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as apphosting from "../../gcp/apphosting";
+
+describe("apphosting", () => {
+  describe("getNextBuildId", () => {
+    let listRollouts: sinon.SinonStub;
+
+    beforeEach(() => {
+      listRollouts = sinon.stub(apphosting, "listRollouts");
+    });
+
+    afterEach(() => {
+      listRollouts.restore();
+    });
+
+    function idPrefix(date: Date): string {
+      const year = date.getUTCFullYear();
+      const month = String(date.getUTCDay()).padStart(2, "0");
+      const day = String(date.getUTCDay()).padStart(2, "0");
+      return `build-${year}-${month}-${day}`;
+    }
+
+    it("Should handle explicit counters", async () => {
```

Comment:
These generally shouldn't be capitalized (though I'm not sure I have any style guidance to point to). When I see this, I read it as a sentence, and "it Should do this" looks abnormal (compared to "it should do this")